core/src/test/scala/com/salesforce/op/evaluators/ExtendedBinaryClassificationMetricsTest.scala

```diff
@@ -0,0 +1,28 @@
+package com.salesforce.op.evaluators
```

Comment:
Header?

Reply:
Good catch! Still barely in negative lines :P